### PR TITLE
Update files page to reflect that PathProvider is now a package

### PR DESCRIPTION
**Pull request by mit-mit**
Update website to match https://github.com/flutter/flutter/pull/9916

### CODE REVIEW

**Comment by mit-mit** [comment]
Failing in code validation:
https://travis-ci.org/flutter/website/builds/230380653
```
/home/travis/build/flutter/website/example/widgets_intro_md_8.dart...
    error • Target of URI doesn't exist: 'package:path_provider/path_provider.dart' at example/reading_writing_files_md_1.dart:5:8 • uri_does_not_exist
  warning • The method 'getApplicationDocumentsDirectory' isn't defined for the class '_FlutterDemoState' at example/reading_writing_files_md_1.dart:39:25 • undefined_method
2 issues found.
(Ran in 6.7s)
```

@sethladd does the code validator work with imports of packages from Pub?

**Comment by sethladd** [comment]
@devoncarew might remember what we need to do to get more imports into the checker.

**Comment by sethladd** [comment]
Ah, I remember. https://github.com/flutter/website/blob/master/tool/travis.sh#L33 is where the tool is run from. The extract.dart is looking for a pubspec.yaml. Maybe we need to add a new dependency to the pubspec.yaml that extract.dart expect?

@devoncarew which pubspec.yaml do we need to edit, if we want to add a dependency for the extract-and-analyze script for the website?

**Comment by devoncarew** [comment]
Will take a look -

**Comment by devoncarew** [comment]
@sethladd, got context back for this repo; @mit-mit, in your PR, add `path_provider: any` to the top-level pubspec.yaml for the repo - the build should go green :)

File: pubspec.yaml [hunk 1]
```diff
@@ -9,3 +9,5 @@ dev_dependencies:
     sdk: flutter
   path: any
   test: ^0.12.13
+  url_launcher:
```
Comment (devoncarew) [inline pubspec.yaml]: I think you need a version here - `any` should do.

Comment (mit-mit) [inline pubspec.yaml]: Hmm, it's working great in several samples like this, I think this is the same as `any`?

**Comment by mit-mit** [comment]
Passing, @sethladd can I get a LGTM?

**Comment by mit-mit** [comment]
Merging to get the website unblocked. TBR @sethladd @szakarias

File: pubspec.yaml (hunk 1 above)
Comment (devoncarew) [inline pubspec.yaml]: (OK, I was unfamiliar w/ that format)

**Pull request merged**